docs/specs/clients/notify/authentication.md

```diff
@@ -175,6 +177,93 @@ Paginated list of notifications with the most recently sent first. Unread notifi
 | Tag     | 4015         |
 | Topic   | notify topic |
 
+## `wc_notifyNotificationsChanged`
```

Comment:
I think we should de-scope this: https://walletconnect.slack.com/archives/C044SKFKELR/p1710787967715679

Reply:
Changed in [3cafcb0](https://github.com/WalletConnect/walletconnect-specs/pull/191/commits/3cafcb0d9895394a7a2e947bb8a351bf5d2a055b)